Instruction.md

```diff
@@ -0,0 +1,4 @@
+### Run instruction:
+1. Install and run XAMPP to create your own DB + tables for testing.
```

Comment:
I would prefer to find a way to use the db in the dockerfile (the same way as Matt) than having to download and creating the DB and tables.

Reply:
WIP.